### DISCUSSION TITLE
Prefetch count in RabbitMQ

### DISCUSSION THREAD

**Discussion by dimbler** [ag2ai/faststream#1253] [Q&A] [answered]
Dear Team, 
Please clarify, how do i can set prefetch_count property, described as channel property?
Could create it only globally, by settings max_consumers in whole broker?
https://aio-pika.readthedocs.io/en/latest/rabbitmq-tutorial/2-work-queues.html

**Comment by Lancetnik** [comment, marked as answer]
Yep, RabbitBroker uses the only one connection and channel, so this option is setted up on broker level by max-consumers option
